kernel/device.c

```diff
@@ -29,7 +29,7 @@ extern u32_t __device_busy_end[];
  * @brief Execute all the device initialization functions at a given level
  *
  * @details Invokes the initialization routine for each device object
- * created by the DEVICE_INIT() macro using the specified level.
+ * created by the DEVICE_INIT(NULL, ) macro using the specified level.
```

Comment:
This change seems to be unwanted as well.

Reply:
Yes. Thank you.

Reply:
Fixed